docs/experiments/experiments/NFSP/JuliaRL_NFSP_KuhnPoker.jl

```diff
@@ -0,0 +1,88 @@
+"""
+NFSP agents trained on Kuhn Poker game.
+"""
```

Comment:
You can follow this format here: https://github.com/JuliaReinforcementLearning/ReinforcementLearning.jl/blob/fc2afa7b9f78c97fc688dbe3aaed9d4d5de366f3/docs/experiments/experiments/CFR/JuliaRL_DeepCFR_OpenSpiel.jl#L1-L7

This format is defined in in [DemoCards](https://johnnychen94.github.io/DemoCards.jl/stable/quickstart/usage_example/julia_demos/1.julia_demo/#juliademocard_example)